### PR TITLE
enhance version flag in Makefile

### PR DESCRIPTION
**Pull request by xiaods**
use alterlative desctibe for flag build

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,11 @@
+# Root directory of the project (absolute path).
+ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```
Comment (vitan) [inline Makefile]: defined but not used

Comment (xiaods) [inline Makefile]: remove it, and rebase again

**Comment by xiaods** [comment]
we also need this patch merged,  PTAL

**Comment by vitan** [comment]
LGTM

**Pull request merged**